timeseries/src/autogluon/timeseries/val_splitter.py

```diff
@@ -0,0 +1,51 @@
+from typing import Iterator, Optional, Tuple
```

Comment:
Why the change in namespace? Aren't these objects just generic splitters? Can they only be used during validation? Are there different splitters for use in training?

Reply:
The only reason for this change is to avoid confusion among new users. 

If `ExpandingWindowSplitter` stays in the `autogluon.timeseries.splitter` namespace and user types `from autogluon.timeseries.splitter import ...` + clicks autocomplete, they will receive suggestions that include deprecated classes (`MultiWindowSplitter`, `LastWindowSplitter`). This seems confusing to me, so I thought about moving all non-deprecated classes to `autogluon.timeseries.val_splitter` and never telling the users about the `autogluon.timeseries.splitter` namespace at all.

Do you think this is a reasonable enough justification?

Reply:
mm, I don't think so. Even if so, I'd put the deprecated stuff to another namespace and remove it later. But I'm OK with just keeping everything in `splitter` too.

Reply:
Fair enough, I agree that having intuitive names for namespaces is more important than autocomplete. Moved all splitter classes back to `autogluon.timeseries.splitter`.